learning/dashboard/angular_routing/Routing/README.md

```diff
@@ -0,0 +1,114 @@
+# ANGULAR
+
+
+Angular is a development platform, built on TypeScript. As a platform, Angular includes:
+
+  A component-based framework for building scalable web applications.
+
+  A collection of well-integrated libraries that cover a wide variety of features, including routing, forms management, client-server communication, and more.
+
+  A suite of developer tools to help you develop, build, test, and update your code.
+
+With Angular, you're taking advantage of a platform that can scale from single-developer projects to enterprise-level applications. Angular is designed to make updating as straightforward as possible, so take advantage of the latest developments with minimal effort.
+
+
+## **Angular Installation** 
+
+
+To install Angular using CLI write this line on your terminal,
+   ``` npm install -g @angular/cli ```
```

Comment:
nit: Single line code blocks can be written with a single backtick. 
Multi-line code blocks can be written with three backticks.